src/include/loggers/settings_logger.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <memory>
+#include "spdlog/sinks/basic_file_sink.h"
+#include "spdlog/spdlog.h"
+
+namespace terrier::settings {
+extern std::shared_ptr<spdlog::logger> settings_logger;
```

Comment:
Why use `shared_ptr` here? Maybe a raw pointer is better.

Reply:
Or `unique_ptr`.

Reply:
Because everyone is using `shared_ptr` in their loggers...